docs/source/tls.rst

```diff
@@ -4,6 +4,8 @@ TLS support
 Receptor supports mutual TLS authentication and encryption for above and below the
 mesh connections.
 
+If you decide to consume TLS authentication in your mesh, the certificates OIDs (1.3.6.1.4.1.2312.19.1) will be verified against the `node.id` specified in the configuration file. If there is no match, the receptor binary will hard exit. You can `Skip Certificate Validation`_ as well.
```

Comment:
This paragraph would fit nicely in the Above the mesh TLS section, as this is the first section that explains the concept of the above the mesh certification.